lib/util/cache/package/redis.ts

```diff
@@ -65,15 +65,19 @@ export async function set(
   // Redis requires TTL to be integer, not float
   const redisTTL = Math.floor(ttlMinutes * 60);
 
-  await client?.set(
-    getKey(namespace, key),
-    JSON.stringify({
-      compress: true,
-      value: await compress(JSON.stringify(value)),
-      expiry: DateTime.local().plus({ minutes: ttlMinutes }),
-    }),
-    { EX: redisTTL }
-  );
+  try {
+    await client?.set(
+      getKey(namespace, key),
+      JSON.stringify({
+        compress: true,
+        value: await compress(JSON.stringify(value)),
+        expiry: DateTime.local().plus({ minutes: ttlMinutes }),
+      }),
+      { EX: redisTTL }
+    );
+  } catch (err) {
+    logger.debug({ err, namespace, key }, 'Error setting cache value');
```

Comment:
This could really blow out the logs of the Redis cache is erroring. Maybe we can reduce it?

Reply:
`logger.once.debug`

Reply:
Will that dedupe bass on the message or the metadata too?

Reply:
Second time this line is hit, it won't print anything even with new metadata.